### PR TITLE
code style updates

### PR DESCRIPTION
**Pull request by colbyfayock**
* Updates the code example to use async/await instead of .then callbacks
* Fixes consistency with formatting
* Removes .env, adds to .gitignore, updates README, and adds .env.example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ This small sample app parallels the interactive demo on the [Cloudinary Doc site
 ### Prerequisites
 
 1. Make sure you have a Cloudinary account or [register for free](https://cloudinary.com/users/register_free).
-2. Copy your `CLOUDINARY_URL` environment variable and replace it in the `.env` file. You can find your credentials in your Cloudinary Console Dashboard.
-2. Make sure you're registered for the [AWS Auto-Tagging add-on](https://cloudinary.com/documentation/google_auto_tagging_addon)  (free tier available).
+2. Create a new `.env` file in the root of the project and copy your `CLOUDINARY_URL` environment variable into it. You can find your credentials in your Cloudinary Console Dashboard.
```
Comment (jackieros) [inline README.md:8]:
```suggestion
2. Rename `.env.example` to `.env` and replace the dummy `CLOUDINARY_URL` environment variable value with yours.  You can copy the full environment variable or the individual credentials from your Cloudinary Console Dashboard. <p>Note that `.env` is included in the .gitignore of this repo. Make sure you never store your Cloudinary environment variable or API secret in any publicly available location. </p>

```

File: src/index.ts [hunk 2]
```diff
@@ -1,61 +1,72 @@
...
+        // Remove background, use a different background image as an underlay, auto-crop to portrait aspect_ratio.
+        // Add conditional text layer if the image has the tag 'overcoat'.
+        // Optimize delivery by resizing and applying auto-format and auto-quality.
+
+        const url = cloudinary.url('bag_model', {
+            transformation: [
+                { effect: 'background_removal' },
+                { underlay: 'buildings_bg', flags: 'relative', width: '1.0', height:'1.0', crop: 'fill' },
+                { gravity: 'auto', crop: 'auto', aspect_ratio: 0.5 },
+                // If the autotagging discovers an "overcoat" and applies the tag to the image during the
```
Comment (jackieros) [inline src/index.ts:52]:
```suggestion
                // If the image has the tag `overcoat` (i.e. if it was added during the autotagging performed during upload), apply a SALE image overlay
```

File: src/index.ts [hunk 3]
```diff
@@ -1,61 +1,72 @@
...
+
 cloudinary.config();
 
+(async function run() {
+    try {
+        // Upload an image from GoogleDrive and apply auto-tagging for all detected categories with >=75% confidence.
+        // Set `overwrite` to true to support running the script multiple times and replacing the same asset.
+        // Use `eager` to warm up the cache with the background-removed variation so delivery can be immediate.
+
+        const results1 = await cloudinary.uploader.upload(
```
Comment (strausr) [inline src/index.ts:16]:
```suggestion
        const uploadResult = await cloudinary.uploader.upload(
```

File: src/index.ts [hunk 4]
```diff
@@ -1,61 +1,72 @@
...
+                public_id: 'bag_model',
+                categorization: 'google_tagging',
+                auto_tagging: 0.75,
+                overwrite: true,
+                // Warming up the cache for transformation effect. For details, see: https://cloudinary.com/documentation/cloudinary_ai_background_removal_addon#removing_the_background_on_the_fly.
+                eager: [{ effect: 'background_removal' }]
+            }
+        );
+
+        console.log('Results:', results1);
```
Comment (strausr) [inline src/index.ts:28]:
```suggestion
        console.log('Upload Results:', uploadResult);
```

File: src/index.ts [hunk 5]
```diff
@@ -1,61 +1,72 @@
...
+
+        // Upload a background image from GoogleDrive for your underlay.
+
+        const results2 = await cloudinary.uploader.upload(
```
Comment (strausr) [inline src/index.ts:32]:
```suggestion
        const backgroundUpload = await cloudinary.uploader.upload(
```

File: src/index.ts [hunk 6]
```diff
@@ -1,61 +1,72 @@
...
+            'https://drive.google.com/uc?export=view&id=15kVDnO77dv5-0hE4-P4g3jKfwPN9ku2U',
+            {
+                public_id: 'buildings_bg',
+                overwrite:true
+            }
+        );
+
+        console.log('Results:', results2);
```
Comment (strausr) [inline src/index.ts:40]:
```suggestion
        console.log('Background Upload Results:', backgroundUpload);
```

File: .env.example [hunk 7]
```diff
@@ -0,0 +1,2 @@
+# Find your Cloudinary URL inside of your Cloudinary Console Dashboard
```
Comment (jackieros) [inline .env.example:1]:
```suggestion
# Find your Cloudinary environment variable URL in your Cloudinary Console Dashboard
```

File: src/index.ts [hunk 8]
```diff
@@ -1,61 +1,72 @@
...
-        },
-        {if: 'end'},
-        { crop: 'scale', height: 450 },
-        { format: 'auto'},
-        { quality: 'auto'}
-    ]
-});
-
-console.log("This is the fully transformed generated url: ", url, "\n");
+        console.log(`Fully transformed generated URL: ${url}`);
```
Comment (jackieros) [inline src/index.ts:68]:
```suggestion
        console.log(`\nThis is the fully transformed generated URL:\n\n${url}\n`);
```

Comment (jackieros) [reply src/index.ts:68]: Suggesting to add some line breaks to make it easier to see in the console.

**Review by jackieros — APPROVED** [review]
@colbyfayock - reviewed together w/ @strausr.  We're both happy with your suggestions. We just dropped in a few minor wording adjustment suggestions. 

Pls accept or comment & then feel free to merge :-)

**Pull request merged**